connection_pool/connection_pool.go

```diff
@@ -98,6 +98,7 @@ type ConnectionPool struct {
 	anyPool    *RoundRobinStrategy
 	poolsMutex sync.RWMutex
 }
+
```

Comment:
It seems like it should be in another commit.

Reply:
Thank you, fixed.